unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Controllers/BIWOutlinerController.cs

```diff
@@ -6,17 +6,18 @@
 
 public interface IBIWOutlinerController
 {
-    public void OutlineEntity(BIWEntity entity);
-    public void CancelEntityOutline(BIWEntity entityToQuitOutline);
-    public void OutlineEntities(List<BIWEntity> entitiesToEdit);
-    public void CheckOutline();
-    public void CancelUnselectedOutlines();
-    public void CancelAllOutlines();
-    public void SetOutlineCheckActive(bool isActive);
+    void OutlineEntity(BIWEntity entity);
+    void CancelEntityOutline(BIWEntity entityToQuitOutline);
+    void OutlineEntities(List<BIWEntity> entitiesToEdit);
+    void CheckOutline();
+    void CancelUnselectedOutlines();
+    void CancelAllOutlines();
+    void SetOutlineCheckActive(bool isActive);
 }
 
 public class BIWOutlinerController : BIWController, IBIWOutlinerController
 {
+    public const int OUTLINER_OPTIMIZATION_TIMES = 10;
```

Comment:
Must this constant be public?

Reply:
In order to access them from the biw outliner test yes, in the future the plan is to expose the private/internals to the test so we don't need it to be public

Reply:
Changed to internal in this PR so we don't forget about this